lib/job.js

```diff
@@ -14,7 +14,13 @@ var Job = module.exports = function Job(args) {
   delete args.agenda;
 
   // Process args
-  args.priority = parsePriority(args.priority) || 0;
+
+  // allow priority overrides (e.g. `job.now('foo', { priority: 'low' })`)
```

Comment:
Yep found this out while debugging some of our tests.

Open an issue? Since we don't have loads of docs on what should or shouldn't be supported I'd say it's a bug.